### PR TITLE
Add full website

### PR DESCRIPTION
**Pull request by rotimiazeez**
# Math Magician App: Full Website 🎉 

> In this project, you will create a full website for the Math magicians app, consisting of several pages and using the components you already created.. 💪🏽 

Additional description of the project and its features. ➕ 🚀  

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Add your own styles to improve the look and feel.
      - In the lesson about styling, we introduced several ways to add styles in a React application:
         > React basic styling (inline styles, importing CSS and pre-processed CSS files).
         > CSS modules.
         > Styled components.
      - In this project, you should use React basic styling.
- For every page, you should create a route.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use React Router to make the routes work.

## Built With ❤️ 

- HTML
- CSS
- React JS

## Deployed Preview 🚀 
[Live Preview](coming soon💯)

### NOTE TO REVIEWER 😉 ❤️  
 😃 👍🏽

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,89 @@
...
+            <nav>
+              <ul style={styles[2]}>
+                <li style={styles[1]}>
+                  <Link style={styles[3]} to="/">Home</Link>
+                </li>
+                <li style={styles[1]}>
+                  <Link style={styles[3]} to="/calculator">Calculator</Link>
+                </li>
+                <li style={styles[1]}>
+                  <Link style={styles[3]} to="/quotes">Quotes</Link>
```
Comment (Davidosky007) [inline src/App.js:59]: * Kindly fix the header section as the heading should be on the right and nav links to the left [Refer to the project general layout](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md#:~:text=The%20general%20layout%20should%20match%20these%20wireframes%3A)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @rotimiazeez 👋 , 

Good job so far! 💪 
* Nice looking readme file ✔️ 
* Well written PR ✔️ 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.js (hunk 1 above)
Comment (rotimiazeez) [reply src/App.js:59]: Done

**Comment by rotimiazeez** [comment]
Hi @Davidosky007,
Thanks for the feedback, I already implemented the changes and hope I get approved this time.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @rotimiazeez, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

tenor (1)


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rotimiazeez** [comment]
@BrianSammit  🙋‍♂️,
Thanks for approving my project 💯.

**Pull request merged**